.github/workflows/test_upload_build_to_test_flight.yml

```diff
@@ -7,6 +7,7 @@ name: Test Upload Build to TestFlight
 ### API_KEY_ID
 ### ISSUER_ID
 ### APPSTORE_CONNECT_API_KEY
+### TEAM_ID
```

Comment:
We don't need to put `TEAM_ID` in secret. Lets add it in `Constant`

Reply:
Ok, so I think I will revert this 😀

Reply:
@ducbm051291 can we remove this comment code for `TEAM_ID`?

Reply:
We still need it :D